uber/config.py

```diff
@@ -120,6 +120,18 @@ def PAGE_PATH(self):
     def PAGE(self):
         return cherrypy.request.path_info.split('/')[-1]
 
+    @property
+    def SUPPORTER_COUNT(self):
+        with sa.Session() as session:
+            attendees = session.query(sa.Attendee)
+            individual_supporters = attendees.filter(or_(sa.Attendee.paid == self.HAS_PAID,
+                                                         sa.Attendee.paid == self.REFUNDED),
+                                                         sa.Attendee.amount_extra == c.SUPPORTER_LEVEL).count()
```

Comment:
You want to say ``>=`` instead of ``==`` here, since someone who registers at above the Supporter level is still a Supporter.  Ditto for a few lines below where you do the same thing.